tests/mock-sl-credentials.js

```diff
@@ -19,9 +19,9 @@ export const slCredentialRevocation = {
     SUITE_CONTEXT_URL
   ],
   id: 'https://example.com/status/1',
-  issuer: 'did:key:z6MkesAkkxuETfHCMdv3gRTKr6iFiQZMjGT5pM8745oSzGX1',
+  issuer: 'did:key:z6MkowtnRyMkCerXvXqjCYUo2mLEeCgX1sWfoP2CZA5UjBop',
   issuanceDate: '2022-06-02T16:00:21Z',
-  type: ['VerifiableCredential', 'BitstringStatusListCredential'],
+  type: [ 'VerifiableCredential', 'BitstringStatusListCredential' ],
```

Comment:
This is an incorrect style change:

```suggestion
  type: ['VerifiableCredential', 'BitstringStatusListCredential'],
```